ar_after_transaction.gemspec

```
@@ -9,7 +9,7 @@ Gem::Specification.new 'ar_after_transaction', ARAfterTransaction::VERSION do |s
   s.homepage = 'http://github.com/grosser/ar_after_transaction'
   s.files = `git ls-files lib Readme.md`.split("\n")
   s.required_ruby_version = '>= 2.6.0'
-  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '< 7.2'
+  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '<= 7.2'
```

we also want to support 7.2.1 etc
```suggestion
  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '< 7.3'
```